client/src/components/VoterRequirements.jsx

```diff
@@ -1,69 +1,175 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import '../static/VoterRequirements.css';
+import React from "react";
+import { Link } from "react-router-dom";
+import "../static/VoterRequirements.css";
 
 const VoterRequirements = () => (
   <div className="VoterRequirements">
-    <a title="See page for author [Public domain], via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File:Vote_with_check_for_v.svg">
-      <img className="horizontal-center" width="250" alt="Vote with check for v" src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Vote_with_check_for_v.svg/256px-Vote_with_check_for_v.svg.png" />
+    <a
+      title="Go to NC Board of Elections"
+      href="https://www.ncsbe.gov/Voters/Registering-to-Vote"
+    >
+      <img
+        className="horizontal-center"
+        width="250"
+        alt="Vote with check for v"
+        src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Vote_with_check_for_v.svg/256px-Vote_with_check_for_v.svg.png"
+      />
     </a>
     <p className="text-large">
-      If you haven’t voted before, or haven’t voted in North Carolina, one of your first questions is probably “Can I Vote?” We can help you answer that!
+      If you haven’t voted before, or haven’t voted in North Carolina, one of
+      your first questions is probably “Can I Vote?” We can help you answer
+      that!
+    </p>
+    <h2 className="VoterRequirements-heading">
+      Step 1: Review Qualifications to Vote
+    </h2>
+    <p>
+      To register to vote in North Carolina, a prospective voter must meet all
+      of the following qualifications:{" "}
```

Comment:
Why does this paragraph end in `{" "}`? This looks like a mistake that should be removed, but maybe I'm wrong and it's some react syntax.

Reply:
As i mentioned in another response, those were added by the linter in places where there was trailing or leading whitespace in an element as react strips that out. In this case because it was back to back <p> elements, it's not needed so I removed it and the whitespace that caused it. Usually I've only had to use that for spacing around mid-string anchor tags. I didn't write the original copy so there must have been some leading/trailing spaces in the paragraphs. I think I got them all that weren't needed.